### ISSUE TITLE
Add the feature of modifying the inserted image size

### ISSUE THREAD

**Issue by dcrozz** [MacDownApp/macdown#677]
I love macdown and use it to take notes every day, but modifying the size of the inserted image is pretty annoying by input `<img src="" width="''>` every time.
If there is a way to implement the syntax `![](pic/location/url =200x) ` like Mou, or just add an alternative way of 'cmd + shift + i' to input '`<img src="" width="''>`'

**Comment by franklinyu** [comment]
The first suggestion is upstream related. It is actually being discussed in [CommonMark](https://talk.commonmark.org/t/support-for-image-dimensions/148). I am not sure whether Hoedown has such an extension; if not, before CommonMark and then Hoedown pick up the change, I don't think we can do anything about this.

The second suggestion of adding a shortcut to insert certain snippet is good, and I prefer this since the produced Markdown is supported more widely. I am more conservative about adding a syntax until a significant percentage of our users require the feature. I think this is a general feature about having a keyboard shortcut to insert any user-defined snippet. We can even have placeholders (like many IDEs).

Note: There is a typo `width="''` which I believe should be `width=""`.

**Comment by dcrozz** [comment]
@franklinyu 
Thank you. I also think the second one is better for I can directly copy the file I create by macdown to the blog that supports markdown syntax such as hexo.
btw. the dummy typo is fixed.

**Comment by uranusjr** [comment]
> I am not sure whether Hoedown has such an extension

No it doesn’t. IIRC the issue has been raised, but never implemented.

**Comment by franklinyu** [comment]
> No it doesn’t. IIRC the issue has been raised, but never implemented.

All right, so the "keyboard shortcut for snippet" is the way to go.

**Comment by uranusjr** [comment]
Assign it to ⌘⇧⌥I maybe? It’s probably better to keep the standard Markdown syntax.

**Comment by franklinyu** [comment]
Yes, <kbd>⌘</kbd> + <kbd>⇧</kbd> + <kbd>⌥</kbd> + <kbd>I</kbd> seems good. Preferably configurable.

**Comment by uranusjr** [comment]
> Preferably configurable.

All hotkeys on Mac (for native applications, at least) are configurable in System Preferences → Keyboard → Shortcuts. :)

**Comment by sjkiss** [comment]
Was this ever implemented?

**Comment by franklinyu** [comment]
@sjkiss No, it has not been implemented yet.

**Comment by sjkiss** [comment]
Some noobs like may end up here and have some difficulty implementing this. 
1. Select System Preferences > Keyboard > Text
2. Add your shortcut and replacement text. 
3. This is crucial: in Macdown, select Edit > Substitutions > Text Replacement. 

Otherwise it won't work.

**Comment by sanjogururaj47** [comment]
Has this been implemented yet? xD
